packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -597,7 +597,9 @@ export class BuildService {
     rootGeneration = true
   ): Promise<CodeGenTypes.DSGResourceData> {
     const resources = await this.resourceService.resources({
-      where: { project: { resources: { some: { id: resourceId } } } }
+      where: {
+        project: { resources: { some: { id: { equals: resourceId } } } }
+      }
```

Comment:
Is that just a refactor or is it a bug fix? 
I think @tupe12334 is working on this exact code right now and it looks like a potential for an unnecessary conflict

Reply:
Done.